.github/workflows/lint.yml

```diff
@@ -41,6 +41,27 @@ jobs:
           # use this.
           warnings: "-native-cc"
 
+  yapf:
+    runs-on: ubuntu-18.04
+    env:
+      BASE_REF: ${{ github.base_ref }}
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        # We have to explicitly fetch the base branch as well
+      - name: Fetching Base Branch
+        run: git fetch --no-tags --prune --depth=1 origin "${BASE_REF?}:${BASE_REF?}"
+      - name: Running yapf
+        run: |
+          python3 -m pip install yapf
+          MODULE_PATH=$(python3 -m pip show yapf | grep Location: | sed -e 's|Location: ||')
+          export PATH="${PATH}:${MODULE_PATH}"
+          chmod +x "${MODULE_PATH}/yapf"
```

Comment:
I'd recommend taking a look at https://stackoverflow.com/q/59207134 for getting a pip module to work

Reply:
See also https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions

which actually appears to be new. Previously I had to do some weirdo "echo" thing (like https://github.com/google/iree/blob/60797421e5316c81211f4d0c1bb00309963521c5/.github/workflows/synchronize_submodules.yml#L37)

Reply:
Thanks for the pointer!